pre_commit/languages/docker.py

```diff
@@ -76,7 +78,28 @@ def install_environment(
         os.mkdir(directory)
 
 
+def _docker_is_rootless() -> bool:  # pragma: win32 no cover
+    returncode, stdout, stderr = cmd_output(
+        'docker', 'system', 'info',
+    )
+    for line in stdout.splitlines():
+        # rootless docker has "rootless"
+        # rootless podman has "rootless: true"
+        if line.strip().startswith('rootless'):
+            if 'false' not in line:
```

Comment:
Note: Technically, the case where `false` is present is not tested. However in practice, we haven't found a case like this, this check is present as precaution.

Reply:
This would be useful as a code comment

Reply:
Is that what is causing the coverage error?

```
Name                             Stmts   Miss Branch BrPart  Cover   Missing
----------------------------------------------------------------------------
pre_commit/languages/docker.py      72      2     10      2    95%   89->91, 91, 101->102, 102
```

Reply:
This is weird. If tests include example outputs containing `rootless`, they should also hit this condition check.

Reply:
@asottile any ideas about this?

Reply:
there's no test for `rootless: false` so line 90 is never hit -- perhaps simpler would be `if line.strip() == 'rootless: true': ...`

Reply:
That would probably not catch docker's behavior. Better to just improve the test matrix